src/components/navigation/z-user-dropdown/styles.css

```diff
@@ -256,8 +263,8 @@ ul > li:last-child {
   }
 
   .useremail {
-    margin-right: calc(var(--space-unit) * 3 + 2px);
-    text-align: end;
+    padding-right: calc(var(--space-unit) * 3 + 2px);
```

Comment:
da abstract mi sembra così

```suggestion
    padding-right: calc(var(--space-unit) * 2);
```

Reply:
Vedi il discorso qui sopra sulle spaziature